zerver/views/registration.py

```diff
@@ -357,28 +378,75 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
-                 'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
-                 'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
-                 'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
-                 }
+        context = {
+            'form': form,
+            'email': email,
+            'key': key,
+            'full_name': request.session.get('authenticated_full_name', None),
+            'lock_name': name_validated and name_changes_disabled(realm),
+            # password_auth_enabled is normally set via our context processor,
+            # but for the registration form, there is no logged in user yet, so
+            # we have to set it here.
+            'creating_new_team': realm_creation,
+            'billing_enabled': billing_enabled,
+            'password_required': password_auth_enabled(realm) and password_required,
+            'require_ldap_password': require_ldap_password,
+            'password_auth_enabled': password_auth_enabled(realm),
+            'root_domain_available': is_root_domain_available(),
+            'default_stream_groups': get_default_stream_groups(realm),
+            'accounts': get_accounts_for_email(email),
+            'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
+            'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
+            'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
+            'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
+        }
     )
 
+def send_email_to_support(realm_name: str, organization_type: str,
+                          repository_link: str,
+                          organization_nature: str,
+                          organization_website: str,
+                          realm_link: str) -> None:
+    if organization_type == "Open-source":
+        send_email(
+            "zerver/emails/organization_info",
+            to_emails=[FromAddress.SUPPORT],
+            from_name="Zulip Account Security",
+            from_address=FromAddress.tokenized_no_reply_address(),
+            context={
+                'realm_name': realm_name,
+                'organization_type': organization_type,
+                'repository_link': repository_link,
+                'realm_link': realm_link,
+            }
+        )
+    elif organization_type == "Non-profit" or organization_type == "Education":
+        send_email(
+            "zerver/emails/organization_info",
+            to_emails=[FromAddress.SUPPORT],
+            from_name="Zulip Account Security",
+            from_address=FromAddress.tokenized_no_reply_address(),
+            context={
+                'realm_name': realm_name,
+                'organization_type': organization_type,
+                'organization_nature': organization_nature,
+                'organization_website': organization_website,
+                'realm_link': realm_link,
+            }
+        )
+    else:
```

Comment:
Ah, just noticed this - I don't think we want to send an email in the "default" case - the purpose of this feature is to automate the Open-Source/Nonprofit application process somewhat.

Reply:
I had removed it. I had previously included it as I thought of always notifying to zulip admin about new organization registration.